### DISCUSSION TITLE
Losing pp should be impossible

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#20792] [Ideas] [answered]
I will preface by saying I am ignorant on whether Lazer scoring is affected by this problem so I would appreciate insight on if this is possible now. Currently in stable, players are able to replace full-combo or near full-combo scores of high pp and high accuracy by deliberately losing accuracy _immediately_ at the start of a map by getting 50s and 100s which are barely weighed by combo compared to the rest of the map. Then, if the rest of the map is combo'd with a higher accuracy than that of the original play, the score is better and thus overwritten.

My idea is to make losing pp programmatically impossible. Once pp is earned, it should not be possible to overwrite it with a lower amount. PP is a record of something you have achieved in osu!. Deliberately losing account pp, known as deranking, is _rampant_ in the non-open rank tournament community. It doesn't make sense as to why this is possible in the game.

Example of how much pp is lost by replacing the score with one of a worse accuracy. Over 150pp was lost from this play alone. Yes - this player has a 94% A-Rank full combo after having 42% accuracy at the first break. What purpose does this being allowed serve?
![firefox_dKVrHwgXe3](https://user-images.githubusercontent.com/38370573/196084600-c5cdd2a3-6de1-4222-99eb-a950f7b69392.png)

https://user-images.githubusercontent.com/38370573/196084453-92615114-353b-4a46-bf84-c67a7bca0087.mp4

These images also show deliberate losses of pp via the same method with the motive of playing in easier rank-restricted tournaments:
![image](https://user-images.githubusercontent.com/38370573/196085089-8cc540a2-b5b4-4d39-badc-1d4eef47fc73.png)
![image](https://user-images.githubusercontent.com/38370573/196085147-3ea140df-38ed-4eb8-b1fa-690cee62566d.png)

**Comment by peppy** [comment, marked as answer]
pp was designed as a secondary ranking system. the primary ranking system is "total score". osu! has a long history so you can probably understand things more by reading up on scoring and pp on the wiki.

there is work underway to make it possible to avoid reducing your pp (either via manual intervention or automatically), but it will take a while more before it comes online.

**Comment by ashandelle** [comment]
Its been over a year now, is there any update on this?

**Reply by peppy** [reply]
Yes, it has been impossible to lose pp on lazer since pp was a thing on lazer.

**Reply by cwillu** [reply]
For what is worth, I just lost something like 126pp on lazer after a single play; possible regression?

**Reply by mixalisnew** [reply]
a bug where people where getting bonus pp from same maps got fixed and people who have many plays on the same maps are a hit affected you can get the pp back from playing more variety

**Comment by Banana090** [comment]
Can you imagine ppy saying "primary ranking system is total score"?
